sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/samples/download/ComposeManifest.groovy

```diff
@@ -0,0 +1,35 @@
+package life.qbic.business.samples.download
+
+/**
+ * <b>Compose a download manifest based on the output of {@link DownloadSamplesOutput}</b>
+ *
+ * <p>This class composes a download manifest based on the output of {@link DownloadSamplesOutput}.</p>
+ *
+ * @since 1.0.0
+ */
+class ComposeManifest implements DownloadSamplesOutput{
+
+    private final ComposeManifestOutput output
+
+    /**
+     * @param output the output receiving the printed manifest
+     * @since 1.0.0
+     */
+    ComposeManifest(ComposeManifestOutput output) {
+        this.output = output
+    }
+
+    @Override
+    void failedExecution(String reason) {
+        output.failedExecution(reason)
+    }
+
+    @Override
+    void foundDownloadableSamples(String projectCode, List<String> sampleCodes) {
+        if (sampleCodes.empty) {
+            output.failedExecution("There is no sample code provided for download.")
+        }
+        DownloadManifest downloadManifest = DownloadManifest.from(sampleCodes)
+        output.readManifest(downloadManifest.print())
```

Comment:
With this line I would assume that downloadManifest.print() handles the printing. But this is not the case, you only transfer the content here

Reply:
I do not understand your comment. Can you elaborate on your assumption, please?

Reply:
A DownloadManifest is a pure core entity.

Reply:
This was just purely related to the print() method. I added a comment for the method itself too because I was looking for what it does

Reply:
I was not meaning to transfer the entity, it was just about the methodnaming

Reply:
How about this 
```suggestion
        output.readManifest(downloadManifest.generateContent())
```

Reply:
no that is too unspecific IMO. It is not clear that this returns a printable representation of the Manifest

Reply:
I think that `read the print` is pretty self-explanatory. `read the generated content` is less clear and expressive

Reply:
I think exactly the other way. But I will not block you